### PR TITLE
Docs: fix typos

### PR DESCRIPTION
**Pull request by ben46**

### CODE REVIEW

File: docs/cli/network.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ View details about the network, like contracts and parameters
 
 ## `celocli network:contracts`
 
-Lists Celo core contracts and their addesses.
+Lists Celo core contracts and their Addesses.
```
Comment (therealharpaljadeja) [inline docs/cli/network.md:12]: Addresses

File: docs/cli/network.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ FLAGS
       --sort=<value>     property to sort by (prepend '-' for descending)
 
 DESCRIPTION
-  Lists Celo core contracts and their addesses.
+  Lists Celo core contracts and their Addesses.
```
Comment (therealharpaljadeja) [inline docs/cli/network.md:33]: Addresses

**Review by ericnakagawa — APPROVED** [review]

**Comment by therealharpaljadeja** [comment]
@ben46 can you fix the conflicts?

**Review by arthurgousset — COMMENTED** [review]
Great finds 💯 The `docs/cli/*` typos need to be fixed in the `celo-org/developer-tooling` repo, because the CLI docs are automatically generated from TS docstrings there, and later sync'ed here. Unless the changes are made there, the typos will be reintroduced when the CLI docs are generated next.

I opened a PR for that there: 
- https://github.com/celo-org/developer-tooling/pull/232

**Pull request closed without merging**